tests/utils.py

```diff
@@ -21,3 +28,67 @@ def run_cli_command(command: str) -> Tuple[str, str, int]:
         command.split(" "), capture_output=True, text=True, check=False
     )
     return result.stdout, result.stderr, result.returncode
+
+
+def cytodataframe_image_display_contains_green_pixels(
```

Comment:
Will we always use green pixels or is there a chance that someday we will let there be a choice?

Reply:
Opened #85 as a result of this.